### DISCUSSION TITLE
[BUG] Not able to build the solution file

### DISCUSSION THREAD

**Discussion by amitkpal83** [nx-dotnet/nx-dotnet#590] [Q&A]
### Current Behavior

Trying to build the project through solution file but it is not happening. Less documentation is there so not sure what wrong i am doing. Have added one solution file through generating the new project through cli and having other while importing the projects. 

### Expected Behavior

Should be able to build the projects through solution file. Able to build the project independently

### Steps to Reproduce

Code is attached. See below the screenshot where i am trying to build the project
[code.zip](https://github.com/nx-dotnet/nx-dotnet/files/10383216/code.zip)

![image](https://user-images.githubusercontent.com/6175985/211565201-d16f2cd6-7606-4a54-b780-ad115c88c8a8.png)

### Nx Report

```shell
.
```

### nx.json

```json
{
  "$schema": "./node_modules/nx/schemas/nx-schema.json",
  "npmScope": "package-based",
  "affected": {
    "defaultBase": "master"
  },
  "tasksRunnerOptions": {
    "default": {
      "runner": "@nrwl/nx-cloud",
      "options": {
        "cacheableOperations": [
          "build",
          "lint",
          "test",
          "e2e"
        ],
        "accessToken": "OTllYmNlNjUtNGVmMC00ZDQ4LWI0ZjgtZTc5ODRjYWY0NWYzfHJlYWQtd3JpdGU="
      }
    }
  },
  "targetDefaults": {
    "build": {
      "dependsOn": [
        "^build"
      ]
    }
  },
  "@nx-dotnet/core:apps": {
    "solutionFile": true,
    "solution": "MySolution.sln"
  },
  "plugins": [
    "@nx-dotnet/core"
  ]
}
```

**Comment by AgentEnder** [comment]
There's not typically a reason to build the actual solution file, and in fact originally the plugin didn't even support using them. What is the use case you are targetting? Currently, you will be unable to build the solution files with our executors / through nx-dotnet, but I don't think it should be needed. All projects that need to be built should be hit when running `nx affected:build`, or similar.

**Comment by amitkpal83** [comment]
@AgentEnder I am having multiple sln files in my project. So when we deploy we use solution file to deploy different application in different locations. That is the my use case and for same reason i was trying to keep multiple solution files to build them separately.

**Comment by AgentEnder** [comment]
I see. You don't generally deploy a solution,  since a solution isn't something that runs. You'd deploy the app project itself
